### ISSUE TITLE
VoiceOver does not read aria-activedescendant

### ISSUE THREAD

**Issue by chrmarti** [electron/electron#14795]
* Output of `node_modules/.bin/electron --version`: v2.0.9 and v3.0.0
* Operating System (Platform and Version): MacOS High Sierra 10.13.6 (17G65)
* Output of `node_modules/.bin/electron --version` on last known working Electron version (if applicable): None

**Expected Behavior**

Whenever `aria-activedescendant` changes the corresponding item should be read. NVDA on Windows does read it. VoiceOver on MacOS does not. Chrome 68 on MacOS does read it.

**Actual behavior**

VoiceOver on MacOS does not read the element referred to by `aria-activedescendant`.

**To Reproduce**

Run the following as the renderer in Electron Fiddle. Place the text cursor in the input field and press up and down keys.

```html
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Hello World!</title>
  </head>
  <body>
  <div id="ex1">
    <div class="combobox-list">
      <div class="group">
        <input id="cb1-input" class="cb_edit" type="text" role="combobox" aria-autocomplete="list"
          aria-haspopup="true" aria-owns="cb1-listbox" aria-expanded="true" placeholder="Choose state">
      </div>
      <div id="cb1-listbox" role="list">
        <div id="lb1-al" role="listitem">Alabama</div>
        <div id="lb1-ak" role="listitem">Alaska</div>
      </div>
    </div>
  </div>

  <script>
    const i = document.getElementById('cb1-input');
    i.addEventListener('keydown', () => {
      const c = i.getAttribute('aria-activedescendant');
      if (c) {
        document.getElementById(c).style.border = null;
      }
      const n = c === 'lb1-al' ? 'lb1-ak' : 'lb1-al';
      i.setAttribute('aria-activedescendant', n);
      document.getElementById(n).style.border = 'solid 1px black';
    });
  </script>
  </body>
</html>
```

**Comment by bpasero** [comment]
@chrmarti can you try with Electron 4? [Download](https://az764295.vo.msecnd.net/exploration/8f0d66e86ff3f6490fa12a0fb61ac913144774cc/VSCode-darwin-exploration.zip)

**Comment by chrmarti** [comment]
@bpasero Indeed, it's fixed in Electron 4. 👍
